library/swift/src/StreamClientBuilder.swift

```diff
@@ -107,6 +108,17 @@ public final class StreamClientBuilder: NSObject {
     return self
   }
 
+  /// Add HTTP filter for requests sent by this client.
+  ///
+  /// - parameter filter: RequestFilter to be invoked for streams.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func addFilter(_ filter: Filter) -> StreamClientBuilder {
```

Comment:
Maybe add a todo here since we'll be switching from instance-based filters in a future PR?

Reply:
Sure thing.